src/main/java/org/openlmis/stockmanagement/domain/BaseEntity.java

```diff
@@ -27,6 +27,7 @@
 @MappedSuperclass
 public abstract class BaseEntity {
   protected static final String TEXT_COLUMN_DEFINITION = "text";
+  protected static final String BOOLEAN_COLUMN_DEFINITION = "bool";
```

Comment:
I think the name here should conform the Postgres one: "boolean"

Reply:
I changed type to 'boolean'.